source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACRBaseTarget.mm

```diff
@@ -6,8 +6,10 @@
 //
 
 #import "ACRBaseTarget.h"
-#import <FluentUI/FluentUI-Swift.h>
 #import <UIKit/UIKit.h>
+#if defined(ADAPTIVECARDS_APP_EXTENSION)
```

Comment:
As a user which consume the AdaptiveCards, what's the impact to them? Is there any document for that: For example, if you want the tooltip, please do this setting on your project.

Should it be a major change from user's aspect?

Reply:
@licanhua   There should be no impact for the consumers. This flag will always exist unless the consumer explicitly exclude the new subspec (UIProviders)

Reply:
by default, users will get tooltip

if they want not to include tooltip, they can choose individual subspecs.


pod 'AdaptiveCards' ~> 'x.x.x' 
get everything.

pick their own, they can do:
pod 'AdaptiveCards/FolderWeNeed1' ~> 'x.x.x'

pod 'AdaptiveCards/FolderWeNeed2' ~> 'x.x.x'

pod 'AdaptiveCards/FolderWeNeed3' ~> 'x.x.x'